.github/workflows/tail-call.yml

```
@@ -114,8 +114,8 @@ jobs:
           find /usr/local/bin -lname '*/Library/Frameworks/Python.framework/*' -delete
           brew install llvm@${{ matrix.llvm }}
           export SDKROOT="$(xcrun --show-sdk-path)"
-          export PATH="/usr/local/opt/llvm/bin:$PATH"
-          export PATH="/opt/homebrew/opt/llvm/bin:$PATH"
+          export PATH="/usr/local/opt/llvm@20/bin:$PATH"
+          export PATH="/opt/homebrew/opt/llvm@20/bin:$PATH"
```

Re-use the variable?
https://github.com/python/cpython/blob/5a7f5ea631aa658f92d651a32d1d064fe9818156/.github/workflows/tail-call.yml#L47-L48

```suggestion
          export PATH="/usr/local/opt/llvm@${{ matrix.llvm }}/bin:$PATH"
          export PATH="/opt/homebrew/opt/llvm@${{ matrix.llvm }}/bin:$PATH"
```

---

I tried this in the past but it makes things more confusing. Basically homebrew on the latest llvm installs it without the version number, but on an older llvm installs it with the version number. Once an LLVM becomes newer homebrew just breaks the old installation by adding a version number to it. So our CI breaks wheneve homebrew decides to update its LLVM.

---

Hmm, right.

* So if we change to latest 21, we need to change `matrix.llvm` from `20` to `21` and two paths to `/llvm/` (whether from `/llvm@20/` or `/llvm{{ matrix.llvm }}/`).

* But if we don't upgrade for a while, and 22 comes out, and we then we only move to 21, then with the variable we'd only need to change `matrix.llvm` from `20` to `21`. Otherwise three changes.

* Similarly if we need to downgrade to 19. One change with a variable, three without.

But I'll leave it to you, which do you prefer? I guess we'd see an error soon enough in the PR without the variable, so no big deal.